steps/github-step-repositories-describe/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM relaysh/core:latest-python
+RUN apk add --no-cache build-base libffi-dev && pip --no-cache-dir install PyGithub
```

Comment:
It can be good practice to uninstall build deps after the build is complete to keep image size down. Here's an example: https://github.com/relay-integrations/relay-ansible/blob/main/steps/playbook-run/Dockerfile#L2-L4

Reply:
lol, I knew you'd ding me for that